docs/source/release/v6.13.0/diffraction.rst

```diff
@@ -2,12 +2,19 @@
 Diffraction Changes
 ===================
 
+.. image:: ../../images/6_13_release/PEARLTransfit-example.png
+   :class: screenshot
+   :width: 385px
+   :align: right
+
```

Comment:
There is nothing to explain what this image is. I can't see a reference to it from any relevant release notes on this page.